docs/OptimizationTips.rst

```diff
@@ -59,7 +59,7 @@ Reducing Dynamic Dispatch
 
 Swift by default is a very dynamic language like Objective-C. Unlike Objective
 C, Swift gives the programmer the ability to improve runtime performance when
-necessary by removing or reducing this dynamicism. This section goes through
+necessary by removing or reducing this dynamism. This section goes through
```

Comment:
Although not in the dictionary, `dynamicism`, is the correct word. It's a combination of the word `dynamic` and `-ism`. See: https://en.wiktionary.org/wiki/-ism